routers/api/v2/tokens.go

```diff
@@ -0,0 +1,65 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	v2 "github.com/unicsmcr/hs_auth/authorization/v2"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"go.uber.org/zap"
+	"net/http"
+	"strings"
+)
+
+// POST: /api/v2/tokens/service
+// x-www-form-urlencoded
+// Request:	 owner string
+//			 allowedURIs string
+//			 expiresAt int64
+// Response: token string
+// Headers:  Authorization <- token
+func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
+	var req struct {
+		Owner       string `form:"owner"`
+		AllowedURIs string `form:"allowedURIs"`
+		ExpiresAt   int64  `form:"expiresAt"`
+	}
+	err := ctx.Bind(&req)
+	if err != nil {
+		r.logger.Debug(err.Error())
```

Comment:
```suggestion
		r.logger.Debug("could not parse request", zap.Error(err))
```
We should ideally use a static string in the log because it makes it easier to search the log file for a specific log (you'd just search for "could not parse request", if we only printed the error message, this approach would not work as the error message is unpredictable)